src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -176,16 +157,8 @@ where
         let mut need_seal_current = false;
         let mut last_range_tombstone_epoch = HummockEpoch::MAX;
         if let Some(builder) = self.current_builder.as_mut() {
-            if is_new_user_key {
-                if switch_builder {
-                    need_seal_current = true;
-                } else if builder.reach_capacity() {
```

Comment:
how to avoid the huge sst if we remove the  `reach_max_sst_size` check?

Reply:
because we do not request `vnode_change`, so it is not necessary to meet the condition `reach_max_sst_size `.